tycho-its/src/test/java/org/eclipse/tycho/test/target/PasswordProtectedP2RepositoryTest.java

```diff
@@ -85,6 +135,8 @@ private Verifier createVerifier(String settingsFile, String settingsSecurityFile
 				new File("projects/target.httpAuthentication/" + settingsFile));
 		Properties systemProperties = verifier.getSystemProperties();
 		systemProperties.setProperty("p2.repo", p2RepoUrl);
+		systemProperties.setProperty("p2.mirror", p2MirrorUrl);
```

Comment:
it is required to modify the existing test-case here?
Can it maybe duplicated into one that uses a mirror and one that do not use a mirror?

Reply:
Do you mean implementing these new tests into a whole new class? I mean, the mirrors are initialized and stopped in common methods (_startServer_ and _StopServer_), so I didn't think adding these two new properties to _createVerifier_ would be a problem. After all, these new properties are only used to resolve their value in the settings file used by the specific test. If they are not present, they do nothing.

Now, could we generalize the _createVerifier_ method so the properties it needs to set are passed as arguments from the individual tests? That might be interesting, even though I am not sure the added complexity would be worth it.

Reply:
By the way, I don't know if you find tests _testMirror_, _testAuthMirror_ and _testTargetDefinitionMirror_ worth merging or if you think I should remove them altogether. After all, the only one that actually reproduces the bug is _testTargetDefinitionAuthMirror_.

Reply:
The system properties containing the mirror URLs (_p2.mirror_ and _p2.authMirror_) are no longer set in the _createVerifier_ method but in the tests themselves. In addition, the _createVerifier_ method has been slightly simplified by using the _setSystemProperty_ method belonging to the _Verifier_ class.